modules/installation-guide/partials/proc_using-configmap-to-configure-the-number-of-workspaces-that-a-user-can-run.adoc

```diff
@@ -1,10 +1,11 @@
-[id="configuring-the-number-of-workspaces-a-user-can-run-using-configmap_{context}"]
-= Configuring the number of workspaces a user can run using ConfigMap
+[id="using-configmap-to-configure-the-number-of-workspaces-that-a-user-can-run_{context}"]
+= Using ConfigMap to configure the number of workspaces that a user can run 
 
-This procedure describes how to run more than one workspace simultaneously using ConfigMap. 
+This procedure describes how to run more than one workspace simultaneously by using ConfigMap. 
 
 .Prerequisites
-* An instance of `{prod-short}` installed using ConfigMap.
+
+* An instance of `{prod-short}` installed by using ConfigMap and the default namespace `{prod-namespace}`.
 Each new workspace creates at least one new mount point to the storage (PV), in write mode. The default `common` storage strategy in `ReadWriteOnce` access mode reuses the same PV to physically store all data from multiple volumes of all workspaces of the same user, and doesn't support multiple mount points. Therefore to increase the number of simultaneously running workspaces, the combination of PVC strategy and access mode must meet the following criteria:
```

Comment:
I would put this information in a note. It's too explanatory for a prerequisite. I suppose the two criteria will need to be listed in the prerequisites nonetheless.